geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/dunit/CqDUnitTest.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.cache.query.cq.dunit;
+
+import static junit.framework.TestCase.assertEquals;
```

Comment:
You should use AssertJ instead. Also, this is the really old JUnit package structure. The newer packaging is `org.junit.**`. Just convert to using `assertThat`:
```
import static org.assertj.core.api.Assertions.assertThat;
```
There's even an IntelliJ plugin called Assertions2Assertj. It's not perfect so you'll want to review the results of what it changes. But it's helpful on big tests.